lib/generator/mocker.ts

```diff
@@ -29,10 +29,16 @@ export default class Mocker {
   }
 
   private generateString(paramSpec: any, paramName: string) {
+    const pwdParams = ["password", "pwd", "adminPassword"];
+
     if (paramSpec.name === "subscriptionId") {
       return uuid.v4().toUpperCase();
     }
 
+    if (pwdParams.includes(paramName)) {
+      return "<a-password-goes-here>";
```

Comment:
This is really great as it will eliminate a whole class of bogus bug bounty requests for things that *look like* leaked passwords but indeed are not valid passwords.  This will make it obvious to everyone that these are not leaked.  Great improvement!